conftest.py

```diff
@@ -1,27 +0,0 @@
-import os
-
-import pytest
-
-
-@pytest.fixture(scope="session", autouse=True)
-def _temp_cwd(tmpdir_factory):
```

Comment:
Puzzling... I'm not sure what's happening there either. I tried testing this locally and that tests (and all other tests) pass without issue for an editable install (installed with `pip install -e .`).

Did the environment previously have asdf installed? Based on the error you saw `AttributeError: module 'asdf' has no attribute 'get_config'` it might be that the pytest plugin of the old installation left behind a 'ghost' namespace (see: https://github.com/asdf-format/asdf/issues/790) but I'm not sure how restoring the `conftest.py` would fix this :-/